net-im/libcommuni/libcommuni-3.5.0.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit qmake-utils
+
+DESCRIPTION="A cross-platform IRC framework written with Qt"
+HOMEPAGE="http://communi.github.io/"
+SRC_URI="https://github.com/communi/libcommuni/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="benchmarks debug examples icu qml static tests +uchardet"
+
+# Compiling with USE="examples" or USE="tests" is currently broken with dev-libs/icu
+REQUIRED_USE="^^ ( icu uchardet )
+		examples? ( uchardet )
+		tests? ( uchardet )"
+
+RDEPEND="dev-qt/qtcore:5
+	dev-qt/qtdeclarative:5
+	icu? ( dev-libs/icu )
+	uchardet? ( app-i18n/uchardet )"
+
+DEPEND="${RDEPEND}"
+
+src_configure() {
+	# Option 'no_rpath' is needed for fixing the QA Notice
+	eqmake5 libcommuni.pro \
+		-config no_rpath
+		-config $(usex benchmarks 'benchmarks' 'no_benchmarks') \
+		-config $(usex debug 'debug' 'release') \
```

Comment:
What does this do? Because it looks suspicious as to another default CFLAGS setter.

Reply:
If I do understand the source code correctly, it only enables some debug messages for output, when using this lib. I didn't saw any new set CFLAGS.